config.json

```diff
@@ -10,6 +10,10 @@
   },
   "blurb": "Scala combines object-oriented and functional programming in one concise, high-level language. Scala's static types help avoid bugs in complex applications, and its JVM and JavaScript runtimes let you build high-performance systems with easy access to huge ecosystems of libraries.",
   "version": 3,
+  "online_editor": {
```

Comment:
According to the [official style guide for Scala 2](https://docs.scala-lang.org/style/indentation.html), these are the correct values

Reply:
👍